packages/neuron-wallet/src/block-sync-renderer/sync/check-and-save/tx.ts

```diff
@@ -50,20 +53,19 @@ export default class CheckTx {
   }
 
   public filterOutputs = (lockHashes: string[]) => {
-    const cells: Cell[] = this.tx.outputs!.map((output, index) => {
+    const cells: Output[] = this.tx.outputs!.map((output, index) => {
```

Comment:
Looking at `filterOutputs` and `filterInputs`, a reader would expect them to return `Output`s and `Inputs`s. 

Question one: is it possible to rename so we know exactly what are these methods doing?
Question two: `filterOuputs` returns outputs, then why the var is called `cells`, not `outputs` (or a generic name as `results`)?

Reply:
1. How about `selectInputsOfMine` / `selectOutputsOfMine` ?
2. I'll change them to `outputs`